src/tour/tour.js

```diff
@@ -194,6 +194,14 @@ angular.module('angular-tour.tour', [])
           scope.ttOffset = parseInt(val, 10) || tourConfig.offset;
```

Comment:
Sounds good. The only use case I could think of where you might want to globally define is if the walkthrough template is very dynamic and nothing is really on the page at compilation, but it's probably not necessary and would be something that a developer would need to specify on each element.

I have the documentation updated locally, updated src/tour/tour.js to use tourConfig.margin and scope.ttMargin, and ran a grunt build to update dist. The tests don't reference offset so they should be fine. I will commit my changes, rebase, and push them out.